openwisp_controller/config/api/filters.py

```diff
@@ -142,3 +143,18 @@ def __init__(self, *args, **kwargs):
 
     class Meta(BaseConfigAPIFilter.Meta):
         model = DeviceGroup
+
+
+class ReversionFilter(BaseConfigAPIFilter):
+    model = filters.CharFilter(field_name="content_type__model", lookup_expr="iexact")
+
+    def _set_valid_filterform_labels(self):
+        self.filters["model"].label = _("Model")
```

Comment:
We are using single quotes for strings, eg: `'model'`. **Please maintain consistency!**